mitmproxy/tools/console/flowview.py

```diff
@@ -117,7 +117,13 @@ def tab_http_request(self):
     def tab_http_response(self):
         flow = self.flow
         assert isinstance(flow, http.HTTPFlow)
-        if self.flow.intercepted and flow.response:
+
+        # prevent renaming when websocket messages are intercepted
+        if (
+            self.flow.intercepted
+            and flow.response
+            and (not flow.websocket or len(flow.websocket.messages) == 0)
```

Comment:
Added this condition because the `Response` tab was renaming itself to `Response intercepted` whenever websocket messages were intercepted